### PR TITLE
Update Home text

### PR DESCRIPTION
**Pull request by matthiasdebaat**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [x] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by zsarnett — DISMISSED** [review]

**Comment by zsarnett** [comment]
https://61e03a25fdbe318ad4116d22--home-assistant-gallery.netlify.app/#concepts/home

**Comment by zsarnett** [comment]
Anchor tags do not work it seems

File: gallery/src/pages/concepts/home.markdown [hunk 1]
```diff
@@ -4,4 +4,21 @@ title: Home
...
 # Welcome to Home Assistant Design
 
-This portal aims to aid designers and developers on improving the Home Assistant interface.
+This portal aims to aid designers and developers on improving the Home Assistant interface. It consists of working code, resources and guidelines.
+
+## Home Assistant interface
+The Home Assistant frontend allows users to browse and control the state of their house, manage their automations and configure integrations. The frontend is designed as a mobile-first experience. It is a progressive web application and offers an app-like experience to our users. The Home Assistant frontend needs to be fast. But it also needs to work on a wide range of old devices.
+
+### Material Design
+The Home Assistant interface is based on Material Design. It's a design system created by Google to quickly build high-quality digital experiences. Components and guidelines that are custom made for Home Assistant are documented on this portal. For all other components please go to <a href="https://material.io" rel="noopener noreferrer target="_blank">material.io</a>.
```
Comment (zsarnett) [inline gallery/src/pages/concepts/home.markdown:13]:
```suggestion
The Home Assistant interface is based on Material Design. It's a design system created by Google to quickly build high-quality digital experiences. Components and guidelines that are custom made for Home Assistant are documented on this portal. For all other components please go to <a href="https://material.io" rel="noopener noreferrer" target="_blank">material.io</a>.
```

File: gallery/src/pages/concepts/home.markdown [hunk 2]
```diff
@@ -4,4 +4,21 @@ title: Home
 
 # Welcome to Home Assistant Design
 
-This portal aims to aid designers and developers on improving the Home Assistant interface.
+This portal aims to aid designers and developers on improving the Home Assistant interface. It consists of working code, resources and guidelines.
+
+## Home Assistant interface
+The Home Assistant frontend allows users to browse and control the state of their house, manage their automations and configure integrations. The frontend is designed as a mobile-first experience. It is a progressive web application and offers an app-like experience to our users. The Home Assistant frontend needs to be fast. But it also needs to work on a wide range of old devices.
```
Comment (balloob) [inline gallery/src/pages/concepts/home.markdown:10]: Not every home is a house :)

```suggestion
The Home Assistant frontend allows users to browse and control the state of their home, manage their automations and configure integrations. The frontend is designed as a mobile-first experience. It is a progressive web application and offers an app-like experience to our users. The Home Assistant frontend needs to be fast. But it also needs to work on a wide range of old devices.
```

File: gallery/src/pages/concepts/home.markdown [hunk 3]
```diff
@@ -4,4 +4,21 @@ title: Home
...
+
+### Material Design
+The Home Assistant interface is based on Material Design. It's a design system created by Google to quickly build high-quality digital experiences. Components and guidelines that are custom made for Home Assistant are documented on this portal. For all other components please go to <a href="https://material.io" rel="noopener noreferrer" target="_blank">material.io</a>.
```
Comment (balloob) [inline gallery/src/pages/concepts/home.markdown:13]:
```suggestion
The Home Assistant interface is based on Material Design. It's a design system created by Google to quickly build high-quality digital experiences. Components and guidelines that are custom made for Home Assistant are documented on this portal. For all other components check <a href="https://material.io" rel="noopener noreferrer" target="_blank">material.io</a>.
```

**Review by balloob — APPROVED** [review]

**Pull request merged**